containers/rust/.devcontainer/devcontainer.json

```diff
@@ -1,33 +1,37 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.158.0/containers/rust
 {
 	"name": "Rust",
 	"build": {
 		"dockerfile": "Dockerfile"
 	},
-	"runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
-
+	"runArgs": [
+		"--cap-add=SYS_PTRACE",
+		"--security-opt",
+		"seccomp=unconfined"
+	],
 	// Set *default* container specific settings.json values on container create.
-	"settings": { 
+	"settings": {
+		"terminal.integrated.shell.linux": "/bin/bash",
```

Comment:
`terminal.integrated.shell.*` has been deprecated.
https://code.visualstudio.com/updates/v1_56#_terminal

I think this line should be rewritten as the below:
```json
		"terminal.integrated.profiles.linux": {
			"bash": {
				"path": "/bin/bash",
			}
		},
		"terminal.integrated.defaultProfile.linux": "bash",
```

Reply:
FYI, in other container definitions, the option was simply removed. #860